### DISCUSSION TITLE
how to setup multiple self-hosted runner on multiple server for single repo ?

### DISCUSSION THREAD

**Discussion by iamnoorsk** [community/community#30758] [Actions] [answered]
how to setup self-hosted runner 1 for test server &  2 for production server

**Comment by vitalykarasik** [comment, marked as answer]
You can launch a few self-hosted runners with different labels: "test", "production", and so on.
And into your workflow you can choose a relevant label.

**Reply by iamnoorsk** [reply]
it worked
thanks
